.env.test

```diff
@@ -20,3 +20,8 @@ BROWSER_TESTS_USER_NAME=
 BROWSER_TESTS_USER_PASSWORD=
 BROWSER_TESTS_JWT_SIGN_SECRET=
 VITE_HELSINKI_PROFILE_URL=
+VITE_MATOMO_URL_BASE="//webanalytics.digiaiiris.com/js/"
+VITE_MATOMO_SITE_ID=813
+VITE_MATOMO_SRC_URL="piwik.min.js"
+VITE_MATOMO_TRACKER_URL="tracker.php"
+VITE_MATOMO_ENABLED="false"
```

Comment:
We have usually used the numbers 1 and 0 for booleans as environment variables..

Reply:
i checked that we used in monorepo true false values so used the same

Reply:
Boolean("false") -> true
Boolean(0) -> false